lib/ui/top_app_bar.dart

```diff
@@ -18,6 +18,7 @@ class TopBarState extends State<TopBar> {
     return AppBar(
         actions:
         <Widget>[
+          Spacer(),
```

Comment:
Addition of this Spacer removed the title of the page for me

Reply:
Tatiana, I believe I just updated this pull request to pull out the spacer to fix the page title.